Kickstarter-iOS/Views/PillsView.swift

```diff
@@ -129,7 +129,7 @@ public final class PillsView: UIView {
         size: size
       )
 
-      x += pillView.frame.maxX + data.interimPillSpacing
+      x += size.width + data.interimPillSpacing
```

Comment:
This fixed an issue where pills weren't taking up the maximum amount of space on a line.